README.md

```diff
@@ -1,13 +1,22 @@
 # myappsample
 repo for javascript
 
-This application is a small myers briggs identifier web app which can be used to identify myers briggs personality types 
+This application is a personality test web app which can be used to identify personality types 
 
 There are a total of 16 types and this application will provide you a small description of each type after it has assessed which type you are trying to identify
 
-Answer all the questions and hit next to find out the myers briggs type 
+It also has sharing functionality enabling useres to share their type to social media
+
+Answer all the questions and hit next to find out your personality type 
 
 *Installation instruction*
 
+The website runs with node on port 8080, this can be changed in the index.js file. 
+
+--To run the website locally--
+
+To run the website locally , download the repository, open the folder MBTI QUIZ, then open the folder called dist and double click the index.html file
+
+--To run the website on node--
 
-To run the website, download the repository, open the folder MBTI QUIZ, then open the folder called dist and double click the index.html file
+open your terminal and run the command node index.js this will set up a server on your localhost on port 8080 in which you can preview the web app
```

Comment:
I would like to see this Readme file in a much more presentable fashion, with marked out steps etc. 

See the following for inspiration: https://medium.com/@meakaakka/a-beginners-guide-to-writing-a-kickass-readme-7ac01da88ab3

Reply:
Added this on Slack, might not have come through https://bulldogjob.com/news/449-how-to-write-a-good-readme-for-your-github-project